compiler/testData/codegen/box/reified/arraysReification/singletonLambda.kt

```diff
@@ -0,0 +1,12 @@
+// IGNORE_BACKEND: JS, WASM
```

Comment:
@pyos Could you move this test to boxInline. It worth to test also an interop of old and new backends

Reply:
Done.

Reply:
Thanks!